ssg/build_yaml.py

```diff
@@ -172,7 +172,7 @@ def dump_yaml(self, file_name, documentation_complete=True):
         if self.extends is not None:
             to_dump["extends"] = self.extends
 
-        if self.platforms is not None:
+        if len(self.platforms) > 0:
```

Comment:
Nitpick: an empty platform evaluates to False:
`if self.platforms:` will work the same.